azurelinuxagent/common/utils/shellutil.py

```diff
@@ -17,16 +17,16 @@
 # Requires Python 2.6+ and Openssl 1.0+
 #
 
-import platform
 import subprocess
+import sys
 
 import azurelinuxagent.common.logger as logger
 from azurelinuxagent.common.future import ustr
 
 
-def get_python_cmd():
-    major_version = platform.python_version_tuple()[0]
-    return "python" if int(major_version) <= 2 else "python{0}".format(major_version)
+def get_complete_python_cmd():
+    # Returns the complete python command (with path) for the current python interpreter
+    return sys.executable
```

Comment:
minor suggestion: I would use sys.executable directly instead of defining this function

Reply:
Done